roles/usegalaxy-eu.remove-orphan-condor-jobs/tasks/main.yml

```diff
@@ -0,0 +1,52 @@
+---
+- name: "Deploy remove-orphan-condor-jobs script"
+  ansible.builtin.copy:
+    src: remove-orphan-condor-jobs
+    dest: "{{ remove_orphan_jobs_bin }}"
+    owner: root
+    group: root
+    mode: 0755
+
+- name: Create cron log directory
+  ansible.builtin.file:
+    state: directory
+    path: "{{ remove_orphan_jobs_log }}"
+    mode: 0755
+    owner: root
+    group: root
+
+- name: Create logfile
+  ansible.builtin.file:
+    state: touch
+    path: "{{ remove_orphan_jobs_log }}"
+    mode: 0664
+    owner: "{{ galaxy_user.name }}"
+    group: "{{ galaxy_group.name }}"
+
+- name: Setup logrotate
+  ansible.builtin.copy:
+    content: |
+      {{ remove_orphan_jobs_log }}
+      {
+          rotate 6
+          daily
+          missingok
+          dateext
+          copytruncate
+          notifempty
+          compress
+      }
+    dest: /etc/logrotate.d/remove-orphan-condor-jobs
+    mode: 0664
+    owner: root
+    group: root
+
+- name: Add to cron
+  ansible.builtin.cron:
+    name: "Remove jobs that are running in condor but unknown to galaxy"
+    day: "*"
+    hour: "*/1"
+    minute: "0"
+    user: "{{ galaxy_user.name }}"
```

Comment:
There might be a permission issue. The `galaxy` user runs the Cron job, while the `root` user owns the log directory.

```
- name: Create cron log directory
  ansible.builtin.file:
    state: directory
    path: "{{ remove_orphan_jobs_log }}"
    mode: 0755
    owner: root
    group: root
```

Reply:
I think `0755` should equal to `-rw-rw-rw-`

Reply:
`0755`  is `rwxr-xr-x`

Reply:
yes that is true, but it is only the directory, the file itself has `644`and is owned by galaxy

Reply:
OK.